src/main/java/jp/co/future/uroborosql/client/command/DescCommand.java

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright (c) 2017-present, Future Corporation
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+package jp.co.future.uroborosql.client.command;
+
+import java.io.PrintWriter;
+import java.io.UnsupportedEncodingException;
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import org.apache.commons.lang3.StringUtils;
+import org.jline.reader.LineReader;
+import org.jline.terminal.Terminal;
+
+import jp.co.future.uroborosql.client.completer.TableNameCompleter;
+import jp.co.future.uroborosql.config.SqlConfig;
+
+/**
+ * Describe table Command
+ *
+ * @author H.Sugimoto
+ */
+public class DescCommand extends ReplCommand {
+	/** DESCで表示する項目 */
+	private static final String[] DESC_COLUMN_LABELS = { "TABLE_NAME", "COLUMN_NAME", "TYPE_NAME", "COLUMN_SIZE",
+			"DECIMAL_DIGITS", "IS_NULLABLE", "COLUMN_DEF", "REMARKS" };
+
+	/**
+	 * Constructor
+	 */
+	@SuppressWarnings("unchecked")
+	public DescCommand() {
+		super(false, TableNameCompleter.class);
+	}
+
+	/**
+	 * {@inheritDoc}
+	 *
+	 * @see jp.co.future.uroborosql.client.command.ReplCommand#execute(org.jline.reader.LineReader, java.lang.String[], jp.co.future.uroborosql.config.SqlConfig, java.util.Properties)
+	 */
+	@Override
+	public boolean execute(final LineReader reader, final String[] parts, final SqlConfig sqlConfig,
+			final Properties props) {
+		PrintWriter writer = reader.getTerminal().writer();
+		writer.println("DESC:");
```

Comment:
@HidekiSugimoto189
In the case of DescCommand, I think that it is unnecessary.

Reply:
OK! fix it.